### ISSUE TITLE
Data source recommendation

### ISSUE THREAD

**Issue by Lextuga007** [nhs-r-community/plot_the_dots#5]
Could the data be fed from the NHSdatasets package?

**Comment by tomjemmett** [comment]
absolutely, we could show some default charts when the user initially loads the package. There is the A&E dataset which could work, or perhaps suggest a new dataset to add to the package?
